pkg/skaffold/jib/jib.go

```diff
@@ -78,3 +100,18 @@ func getDependencies(cmd *exec.Cmd) ([]string, error) {
 	sort.Strings(deps)
 	return deps, nil
 }
+
+// relativize tries to make path relative to the root location(s)
+func relativize(path string, roots ...string) (string, error) {
+	if !filepath.IsAbs(path) {
+		return path, nil
+	}
+	dotDotSlash := fmt.Sprintf("..%c", filepath.Separator)
```

Comment:
I wonder if `dotDotSlash` can be put in a const variable?

Reply:
I had hoped so too but the compiler complained

Reply:
Aha: `const dotDotSlash = ".." + string(filepath.Separator)`